### ISSUE TITLE
No such file or directory sh

### ISSUE THREAD

**Issue by QACodeDev** [BushnevYuri/DockerGrafanaInfluxKit#5]
I installed docker and cloned the grafana influx repository but with the command docker-compose up, I am getting the error -
Starting dockergrafanainfluxkit_influxdb_1 ... done
Starting dockergrafanainfluxkit_grafana_1 ... done
Attaching to dockergrafanainfluxkit_influxdb_1, dockergrafanainfluxkit_grafana_1
': No such file or directorycute 'sh
: No such file or directory sh
dockergrafanainfluxkit_influxdb_1 exited with code 127
dockergrafanainfluxkit_grafana_1 exited with code 127

**Comment by darpandodiya** [comment]
Copying from one of the comments on the article:

"Got the same error.

To resolve it I just opened docker file and entrypoint file for grafana and influxdb in the notepad++
and changed from Windows to Unix system and save"

This worked for me too

**Comment by fortanu82** [comment]
Yes, I also faced the same issue and by converting the files from dos2unix command worked.
i.e install a tool on your o/s (windows/mac). I have windows system this I executed run the command "dos2unix -o filename" to make it work.

**Comment by Maithrie** [comment]
If its a windows machine, shouldnt it be unix2dos?

**Comment by Maithrie** [comment]
Even when i use dos2unix, it gives below error messages.

docker-compose up
Container gatling-grafana-influxdb-influxdb-1  Created
Container gatling-grafana-influxdb-grafana-1  Created
Attaching to gatling-grafana-influxdb-grafana-1, gatling-grafana-influxdb-influxdb-1
': No such file or directoryluxdb-1  | env: can't execute 'sh
gatling-grafana-influxdb-influxdb-1 exited with code 127
gatling-grafana-influxdb-grafana-1   | /usr/bin/env: 'sh\r': No such file or directory
gatling-grafana-influxdb-grafana-1 exited with code 127
